plugins/foreman_maintain/0.2/index.md

```diff
@@ -0,0 +1,405 @@
+| Getting dump of Mongo DB                                            [OK]      
+--------------------------------------------------------------------------------
+Backup Postgres global objects online:                                [OK]
+--------------------------------------------------------------------------------
+Backup Candlepin database online: 
+\ Getting Candlepin DB dump                                           [OK]      
+--------------------------------------------------------------------------------
+Backup Foreman database online: 
+- Getting Foreman DB dump                                             [OK]      
+--------------------------------------------------------------------------------
+Compress backup data to save space:                                   [OK]
+--------------------------------------------------------------------------------
+
+Done with backup: 2018-05-03 03:20:42 +0000
+**** BACKUP Complete, contents can be found in: /tmp/some/dir/katello-backup-2018-05-03-03-20-35 ****
+```
+You can see the metadata step was skipped.
+
+To see a list of available step labels use:
+```
+foreman-maintain advanced procedure run -h
+``` 
+
+In some cases it make sense to run one individual step, e.g. collect config files for debugging purposes:
+```
+# foreman-maintain advanced procedure run backup-config-files --backup-dir /tmp/backup
+Running ForemanMaintain::Scenario
+================================================================================
+Backup config files: 
+\ Collecting config files to backup                                   [OK]      
+--------------------------------------------------------------------------------
+```
+ 
+### Directory names
+By default the backup is stored in subdirectory named `<katello|foreman|satellite|capsule>-backup-YYYY-MM-DD-hh-mm-ss` in a directory provided on a command line.
+
+If you need to set the directory name yourself use switch `--preserve-directory` and the backup will be stored in the directory you provide on the command line. Be aware that user `postgres` needs write access to that directory if you have local PgSQL database.
+
+```
+foreman-maintain backup online -y --preserve-directory /tmp/my_backup_dir
+Starting backup: 2018-05-03 04:03:22 +0000
+Running preparation steps required to run the next scenarios
+================================================================================
+... LIST SHORTENED ...
+
+Done with backup: 2018-05-03 04:03:41 +0000
+**** BACKUP Complete, contents can be found in: /tmp/my_backup_dir ****
+```
+Note there was no subdir created. 
+
+Also note that when using `--preserve-directory` no data are removed from it when the backup fail.
+
+### Offline backup
+Offline backup is safe way to backup your server for later restoring. All the services are down during the backup and the instance is in a maintenance mode thus not accessible from outside. The instance is started back into normal after the backup finished.
```

Comment:
`is the safe way`...`for later restoration`
`The instance is started normally after the backup completes.`